colin/core/fmf_extension.py

```diff
@@ -50,20 +50,26 @@ def references(self, datatree, whole=False):
 
         :param whole: 'whole' param of original climb method, in colin this is not used anyhow now
                       iterate over all items not only leaves if True
-        :param datatree: original tree with testcases to contain parent nodes
+        :param datatrees: dict of original trees with testcases to contain parent nodes
```

Comment:
since it's a dict, it would be nice to provide a short example; especially what are the keys